### PR TITLE
[cpuinfo] updated to last commit from 2022-07-19

### PR DESCRIPTION
**Pull request by vladimir-aubrecht**
**Describe the pull request**
Cpuinfo updated to last commit from 2022-07-19

- #### What does your PR fix?
  New version is adding win-arm64 support.

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
  all, No

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
  Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
  Yes

### CODE REVIEW

File: ports/cpuinfo/vcpkg.json [hunk 1]
```diff
@@ -1,11 +1,10 @@
 {
   "name": "cpuinfo",
-  "version-date": "2022-04-02",
-  "port-version": 1,
+  "version-date": "2022-07-19",
+  "port-version": 2,
```
Comment (Neumann-A) [inline ports/cpuinfo/vcpkg.json:4]:
```suggestion
```

if you update `version: x` you delete `port-version`

File: ports/cpuinfo/vcpkg.json [hunk 2]
```diff
@@ -1,11 +1,10 @@
 {
   "name": "cpuinfo",
-  "version-date": "2022-04-02",
-  "port-version": 1,
+  "version-date": "2022-07-19",
   "description": "CPU INFOrmation library (x86/x86-64/ARM/ARM64, Linux/Windows/Android/macOS/iOS)",
   "homepage": "https://github.com/pytorch/cpuinfo",
   "license": "BSD-2-Clause",
-  "supports": "!(arm & windows)",
+  "supports": "!(uwp & arm32)",
```
Comment (JackBoosY) [inline ports/cpuinfo/vcpkg.json:7]: It seems that this port support uwp before, what's going wrong on this version?

Comment (vladimir-aubrecht) [reply ports/cpuinfo/vcpkg.json:7]: Actually it did support UWP only on non-arm platforms.
This version added arm64 support, which wasn't supported before and which also supports UWP now.
What is not supported (and wasn't supported) is UWP on arm32 only as code of the library has unfortunately written condition which checks if it's arm64 else do x86/x64 branch 😕

**Review by JackBoosY — DISMISSED** [review]

**Comment by JackBoosY** [comment]
Already tested all features successsfully on x86-windows and x64-windows-static.

File: versions/c-/cpuinfo.json [hunk 3]
```diff
@@ -1,5 +1,15 @@
...
+    {
+      "git-tree": "5ee1eff9fe6cfa584b697e71ce7081a69ef2c51d",
+      "version-date": "2022-07-19",
+      "port-version": 0
+    },
+    {
+      "git-tree": "27de98dc25f757fa2e5003b6f4c4ae38a0e31120",
+      "version-date": "2022-07-19",
+      "port-version": 2
+    },
```
Comment (Thomas1664) [inline versions/c-/cpuinfo.json:12]:
```suggestion
    {
      "git-tree": "5ee1eff9fe6cfa584b697e71ce7081a69ef2c51d",
      "version-date": "2022-07-19",
      "port-version": 0
    },
```

You may add at most only one version.

**Review by Thomas1664 — CHANGES_REQUESTED** [review]

**Review by JackBoosY — APPROVED** [review]

**Comment by ras0219-msft** [comment]
LGTM, thanks!

**Pull request merged**